src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Rectangle extends Figure {
+    private int sideA;
+    private int sideB;
+
+    public Rectangle() {
+        super();
+        this.sideA = new Random().nextInt(100);
+        this.sideB = new Random().nextInt(100);
```

Comment:
Random value generation should be handled in the `FigureSupplier` class, as per the checklist: 'You can pass random values to the constructor of a figure instead of generating them inside figure classes.'